plugins/plugins/frontend/public/javascripts/countly.views.js

```diff
@@ -871,6 +928,19 @@ window.ConfigurationsView = countlyView.extend({
                 }
             });
 
+            $('body').off('click').on('click', function(e) {
```

Comment:
And here we need some control mechanism, not removing all click listeners

Reply:
Assuming you need this for hiding popup, you can try using blur event like we do in other places:
https://github.com/Countly/countly-server/blob/master/frontend/express/public/javascripts/countly/countly.helpers.js#L1119-L1123